Signal/src/call/CallService.swift

```diff
@@ -1028,21 +1060,33 @@ protocol CallServiceObserver: class {
 
         Logger.debug("\(TAG) in \(#function)")
 
-        PeerConnectionClient.stopAudioSession()
-        peerConnectionClient?.terminate()
-
-        peerConnectionClient = nil
         localVideoTrack = nil
         remoteVideoTrack = nil
         isRemoteVideoEnabled = false
+
+        // We make a local copy of peerConnectionClient and then clear
+        // the peerConnectionClient property immediately. This class' 
+        // PeerConnectionClientDelegate methods exit early if their 
+        // peerConnectionClient argument doesn't match the peerConnectionClient
+        // property.
+        //
+        // In practice this won't matter, since PeerConnectionClient invokes
+        // its delegate methods asynchronously, but we don't want to bake that
+        // assumption into this logic.
+        var peerConnectionClientCopy = peerConnectionClient
```

Comment:
Okay, I've removed this. As noted, I was just trying to future-proof us against later changes to PCC, but better not to include anything confusing.